rules/Php73/Rector/FuncCall/RegexDashEscapeRector.php

```diff
@@ -21,6 +21,13 @@
  */
 final class RegexDashEscapeRector extends AbstractRector
 {
+
+    /**
+     * @var string
+     * @see https://regex101.com/r/1e3hoF/1
+     */
+    private const THREE_BACKSLASH_FOR_ESCAPE_NEXT_REGEX = '#(?<=[^\\\\])\\\\{3}(?=[^\\\\])#';
```

Comment:
it seems working on regex101.com https://regex101.com/r/iQbGgZ/1/ but somehow not working on tests, possibly need escape multiple `\` as `\\\\`

Reply:
fixed, updated to use `{2}` as detected only 2 after `$this->regexPatternArgumentManipulator->matchCallArgumentWithRegexPattern()` call